test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -761,3 +761,202 @@ Feature: Bind values from a secret referred in backing service resource
             """
             bla
             """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+          """
+        Then Service Binding "bind-provisioned-service-1" is ready
+        And jq ".status.secret" of Service Binding "bind-provisioned-service-1" should be changed to "provisioned-secret-1"
+        And Content of file "/bindings/bind-provisioned-service-1/username" in application pod is
+            """
+            foo
+            """
+        And Content of file "/bindings/bind-provisioned-service-1/password" in application pod is
+            """
+            bar
+            """
+
+    Scenario: Inject binding to an application from a Secret resource referred as service with mappings
+        Given The Secret is present
+            """
+            apiVersion: v1
+            kind: Secret
+            metadata:
+                name: provisioned-secret-1
+            stringData:
+                username: foo
+                password: bar
+            """
+        * Generic test application "myaop-provision-srv" is running
+        When Service Binding is applied
+          """
+          apiVersion: binding.operators.coreos.com/v1alpha1
+          kind: ServiceBinding
+          metadata:
+              name: bind-provisioned-service-1
+          spec:
+              bindAsFiles: true
+              services:
+              - group: ""
+                version: v1
+                kind: Secret
+                name: provisioned-secret-1
+                id: sec
+              application:
+                name: myaop-provision-srv
+                group: apps
+                version: v1
+                resource: deployments
+              mappings:
+                - name: username_with_password
+                  value: '{{ .username }}:{{ .password }}'
+          """
+        Then Service Binding "bind-provisioned-service-1" is ready
+        And jq ".status.secret" of Service Binding "bind-provisioned-service-1" should starts with "bind-provisioned-service-1"
```

Comment:
we should not advertise this kind of info to users - it is not a part of api/contract - the spec only says that this field will be populated with the intermediate secret name, but does not say anything about naming convention.

Reply:
Done. 832c72fcec87d9f227ff5ea071ff759a112edc95